.circleci/config.yml

```diff
@@ -373,12 +373,12 @@ jobs:
     steps:
```

Comment:
the reason i had to revert the above lines is that this config is using a machine and not a docker image, in circle ci they only have machines that are `classic` and there is no well-maintained image of docker that contains docker and node, so its either we do an image for this use case or just keep it as is @melekes